File: 100-Days-Series/Day 93/FindElementInRange.js

// Function to number in a particular range.
const getFindNumberInRange=(arr,sr,er,number)=>{
    if(sr<0 || sr>=arr.length || sr>er){
        return -1;
    }
    for(let i=sr;i<=er;i++){
        if(arr[i]==number){
            return i;
        }
    }
    return -1;
}
// Let's Start Here
const arr=[1,2,3,4,5,6,7,8,9,10];
let sr=5;
let er=8;
let numerbToFind=6;
let res=getFindNumberInRange(arr,sr,er,numerbToFind);
if(res!=-1){
    console.log("Found at Index "+res);
}else{
    console.log("Not Found!!");
}

File: 100-Days-Series/Day 93/FindElementInRange.java
public class FindElementInRange {
    // Method to find number in a particular range.
    public static int getFindNumberInRange(int arr[],int sr,int er,int number){
        if(sr<0 || er>=arr.length || sr>er){
            return -1;
        }
        for(int i=sr;i<=er;i++){
            if(arr[i]==number){
                return i;
            }
        }
        return -1;
    }
    // Main Method
    public static void main(String [] args){
        int arr[]={1,2,3,4,5,6,7,8,9,10};
        int sr=2;
        int er=5;
        int numberToFind=3;
        int res=getFindNumberInRange(arr, sr, er, numberToFind);
        if(res!=-1){
            System.out.println("Found at Index "+res);
        }else{
            System.out.println("Not Found!!");
        }
    }
}
